serializers/src/glTF/2.0/glTFExporter.ts

```diff
@@ -729,14 +738,57 @@ export class _Exporter {
                 }
                 break;
             }
+            case VertexBuffer.MatricesIndicesKind:
+            case VertexBuffer.MatricesIndicesExtraKind: {
+                for (let k = 0, length = meshAttributeArray.length / stride; k < length; ++k) {
+                    index = k * stride;
+                    const vertexData = Vector4.FromArray(meshAttributeArray, index);
+                    vertexAttributes.push(vertexData.asArray());
+                }
+                break;
+            }
+            case VertexBuffer.MatricesWeightsKind:
+            case VertexBuffer.MatricesWeightsExtraKind: {
+                for (let k = 0, length = meshAttributeArray.length / stride; k < length; ++k) {
+                    index = k * stride;
+                    const vertexData = Vector4.FromArray(meshAttributeArray, index);
+                    vertexData.normalize();

```

Comment:
Current outstanding issue is that currently exporting test scene: https://playground.babylonjs.com/#T087A8#12

Exports with errors related to non-normalized skinning weights...
I added this `normalize()` call @L755 to address this, but the error persists. I'll continue to investigate....

Reply:
Duh. I'm currently normalizing a vec4 using euclidean distance... we just want all the members to add up to 1. I should have a fix shortly...

Reply:
fixed.